.github/workflows/continuous-integration.yml

```diff
@@ -152,7 +212,364 @@ jobs:
+          - collation: "Latin1_General_100_CS_AS"
+            php-version: "7.4"
+            driver: "sqlsrv"
+          - collation: "Latin1_General_100_CS_AS"
+            php-version: "7.4"
+            driver: "pdo-sqlsrv"
+
+    services:
+      mssql:
+        image: "microsoft/mssql-server-linux:2017-latest"
+        env:
+          ACCEPT_EULA: "Y"
+          SA_PASSWORD: "Doctrine2018"
+          MSSQL_COLLATION: "${{ matrix.collation }}"
+
+        options: >-
+          --health-cmd "echo quit | /opt/mssql-tools/bin/sqlcmd -S 127.0.0.1 -l 1 -U sa -P Doctrine2018"
+
+        ports:
+          - "1433:1433"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+        with:
+          fetch-depth: 2
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          php-version: "${{ matrix.php-version }}"
+          coverage: "pcov"
+          tools: "pecl"
+          extensions: "${{ matrix.driver }}-5.7.0preview"
+
+      - name: "Cache dependencies installed with composer"
+        uses: "actions/cache@v2"
+        with:
+          path: "~/.composer/cache"
+          key: "php-${{ matrix.php-version }}-composer-locked-${{ hashFiles('composer.lock') }}"
+          restore-keys: "php-${{ matrix.php-version }}-composer-locked-"
+
+      - name: "Install dependencies with composer"
+        run: "composer install --no-interaction --no-progress --no-suggest"
+
+      - name: "Run PHPUnit"
+        run: "vendor/bin/phpunit -c ci/github/phpunit/${{ matrix.driver }}.xml --coverage-clover=coverage.xml"
+
+      - name: "Upload Code Coverage"
+        uses: "codecov/codecov-action@v1"
+
+  phpunit-ibm-db2:
+    name: "PHPUnit with IBM DB2"
+    runs-on: "ubuntu-18.04"
+    needs: "phpunit-smoke-check"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.3"
+
+    services:
+      ibm_db2:
+        image: "ibmcom/db2:11.5.0.0"
+        env:
+          DB2INST1_PASSWORD: "Doctrine2018"
+          LICENSE: "accept"
+          DBNAME: "doctrine"
+
+        options: "--privileged=true"
+
+        ports:
+          - "50000:50000"
+
+    steps:
+      - name: "Perform healthcheck from the outside"
+        run: "docker logs -f ${{ job.services.ibm_db2.id }} | sed '/(*) Setup has completed./ q'"
+
+      - name: "Create temporary tablespace"
+        run: "docker exec ${{ job.services.ibm_db2.id }} su - db2inst1 -c 'db2 CONNECT TO doctrine && db2 CREATE USER TEMPORARY TABLESPACE doctrine_tbsp PAGESIZE 4 K'"
+
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+        with:
+          fetch-depth: 2
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          php-version: "${{ matrix.php-version }}"
+          coverage: "pcov"
+          ini-values: "extension=ibm_db2.so, ibm_db2.instance_name=db2inst1"
+
+      - name: "Install ibm_db2 extension"
+        run: "ci/github/install-db2-ibm_db2.sh ${{ matrix.php-version }}"
```

Comment:
This script is named this weirdly because the name was autogenerated on Travis from the build matrix parameters. Let's rename it to `ext/install-ibm-db2.sh`.

Reply:
Done and undone

Reply:
It still should be `ext/install-ibm_db2.sh`.